src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -101,22 +127,36 @@ export class SchoolSelectField extends React.Component {
   render() {
     const {
       currentPageNumber,
+      errorMessages,
       facilityCodeSelected,
       formContext,
       institutionQuery,
       institutions,
       institutionSelected,
+      manualSchoolEntryChecked,
       pagesCount,
       searchInputValue,
       searchResultsCount,
+      showErrors,
       showInstitutions,
       showInstitutionsLoading,
       showNoResultsFound,
       showPagination,
       showPaginationLoading,
+      showSearchResults
     } = this.props;
 
-    const fieldsetClass = classNames('search-select-school-fieldset');
+    const fieldsetClassNames = classNames('search-select-school-fieldset');
+    const schoolSearchClassNames = classNames('school-search', {
+      'usa-input-error': showErrors
+    });
+    const labelClassNames = classNames('school-search-label', {
+      'usa-input-error-label': showErrors
+    });
+
+    if (formContext.reviewMode && manualSchoolEntryChecked) {
+      return (<div></div>);
```

Comment:
The form generator is rendering the `SchoolSelectField` on the review page so it has to hide itself if manual entry is selected.